src/pages/search/SearchResults.tsx

```diff
@@ -12,14 +12,18 @@ const StyledSearchResults = styled.div`
   padding-bottom: 1rem;
 `;
 
+const StyledContributor = styled.span`
+  padding-right: 1rem;
+`;
```

Comment:
Jeg får ikke opp noen contributors etter søk
![bilde](https://user-images.githubusercontent.com/6313468/93094411-2db20e00-f6a2-11ea-9b19-c0e2c6fd36a6.png)

Reply:
^ Når jeg trykker på en av de 3 treffene der blir jeg sendt til http://localhost:3000/publication/undefined/public

Reply:
det der er en gammel publikasjon som ikke er gyldig lengre (ref det jeg sa på standup. Noen publikasjoner matcher ikke datamodellen lengre). Gyldig modell ser sånn ut:
```
export interface SearchResult {
  id: string;
  contributor: SearchResultContributor[];
  date: string;
  owner: string;
  title: string;
}
```

Reply:
jeg opprettet denne nå, men den har `contributors`
![bilde](https://user-images.githubusercontent.com/6313468/93172655-8f6a8a80-f72b-11ea-94c4-8d2158991b7d.png)
1970 blir forresten til 1.1.1970 også, der har vi en liten feil som sikkert dukker opp flere plasser 😬

Reply:
ser at den der også har identifier isteden for id. kanskje det er noe forskjell på om de blir opprettet via doi eller fil? mulig bug på backend kanskje

Reply:
var visst ikke helt ferdig på backend, men det blir hetende `id` og `contributors`. Jeg oppdaterer. Må fikse dato-bugen også

Reply:
Rurik fikser slik at vi får denne strukturen tilbake på date: `{date: {year: 2020, month: 11, day: 22}}`

Reply:
👍  noen publikasjoner har bare år også, så da trenger vi ikke full dato